packages/cli/src/deploy/ensureWorldFactory.ts

```diff
@@ -11,10 +11,7 @@ import { salt } from "./common";
 import { ensureContractsDeployed } from "./ensureContractsDeployed";
 import { Contract } from "./ensureContract";
 
-export async function ensureWorldFactory(
-  client: Client<Transport, Chain | undefined, Account>,
-  deployerAddress: Hex,
-): Promise<Address> {
+export function getWorldFactoryContracts(deployerAddress: Hex): readonly Contract[] {
```

Comment:
can we move this into its own file outside of deploy since it's used in verify now too?

Reply:
Yes we should be consistent with one function-per-file!